docs/guides/backends/local/index.md

```diff
@@ -14,264 +14,141 @@ section_menu_id: guides
-| `local.VolumeSource` | `Required` | Any Kubernetes volume. Can be specified inlined. Example: `hostPath`.                                                                                                                                                                    |
+> Note that by default, KubeStash run an initializer job for the local backend, which doesn’t have file write permission. So, in order to achieve
+that you must give file system group permission, achieved by specifying `spec.runtimeSettings.pod.securityContext.fsGroup` in the `BackupStorage` configuration.
 
-Here, we are going to show some sample `Repository` crds that uses different Kubernetes volume as a backend.
+Here, we are going to show some sample `BackupStorage` objects that uses different Kubernetes volume as a backend.
 
-##### HostPath volume as Backend
+### HostPath volume as Backend
 
-Below, the YAML of a sample `Repository` crd that uses a `hostPath` volume as a backend.
+Below, the YAML of a sample `BackupStorage` object that uses a `hostPath` volume as a backend.
 
 ```yaml
-apiVersion: stash.appscode.com/v1alpha1
-kind: Repository
+apiVersion: storage.kubestash.com/v1alpha1
+kind: BackupStorage
 metadata:
-  name: local-repo-with-hostpath
+  name: local-storage-with-hostpath
   namespace: demo
 spec:
-  backend:
+  storage:
+    provider: local
     local:
       mountPath: /safe/data
       hostPath:
-        path: /data/stash-test/repo
-    storageSecretName: local-secret
+        path: /data/kubestash-test/storage
+  usagePolicy:
+    allowedNamespaces:
+      from: All
+  default: true
+  deletionPolicy: WipeOut
+  runtimeSettings:
+    pod:
+      securityContext:
+        runAsUser: 0
 ```
 
-Create the `Repository` we have shown above using the following command,
+Create the `BackupStoage` we have shown above using the following command,
 
 ```bash
 $ kubectl apply -f https://github.com/kubestash/docs/raw/{{< param "info.version" >}}/docs/guides/backends/local/examples/hostPath.yaml
-repository/local-repo-with-hostpath created
+backupstorage.storage.kubestash.com/local-storage-with-hostpath created
 ```
 
->Note that by default, Stash runs as `non-root` user. `hostPath` volume is writable only for `root` user. So, in order to use `hostPath` volume as backend, either you have to run Stash as `root` user using securityContext or you have to change the permission of the `hostPath` to make it writable for `non-root` users.
+> Note that by default, Kubestash run `BackupStorage` initializer job with a `non-root` user. `hostPath` volume is writable only for the `root` user.
+So, in order to use `hostPath` volume as a backend, you must either run initializer job as the `root` user, achieved by specifying 
+`spec.runtimeSettings.pod.securityContext.runAsUser` in the `BackupStorage` configuration, or adjust the permissions of the `hostPath` to allow write access for `non-root` users.
```

Comment:
```suggestion
> Since a `hostPath` volume is typically writable only by the root user, you'll need to either run the initializer job as the `root` user or modify permissions directly on the host filesystem to enable non-root write access. 
```